models/product-tailoring/src/product-tailoring-data/fields-config.ts

```diff
@@ -34,6 +34,7 @@ export const graphqlFieldsConfig: TModelFieldsConfig<TProductTailoringDataGraphq
       metaDescriptionAllLocales: null,
       metaKeywordsAllLocales: null,
       __typename: 'ProductTailoringData',
+      attributesRaw: null,
```

Comment:
This attribute is not defined as optional in the corresponding type so we should be populating it at least with an empty array.